client/analytics/components/report-table/index.js

```diff
@@ -323,7 +323,7 @@ const ReportTable = ( props ) => {
 	const isLoading =
 		isRequesting || tableData.isRequesting || primaryData.isRequesting;
 	const totals = get( primaryData, [ 'data', 'totals' ], {} );
-	const totalResults = items.totalResults;
+	const totalResults = items.totalResults || 0;
```

Comment:
`totalResults` will be 0 by default in order to avoid a JS warning in the `Analytics` > `Orders` view.